### PR TITLE
Left button support - Clean up

### PR DESCRIPTION
**Pull request by Fogh**
Clean up for the left button PR - #33

### CODE REVIEW

File: MessageViewController/MessageView.swift [hunk 1]
```diff
@@ -110,50 +110,30 @@ public final class MessageView: UIView, MessageTextViewListener {
...
 
     public func set(buttonIcon: UIImage?, for state: UIControlState, type: buttonType) {
         switch type {
-            case .left:
-                setLeft(buttonIcon: buttonIcon, for: state)
-                break
-            case .right:
-                setRight(buttonIcon: buttonIcon, for: state)
-                break
+        case .left:
```
Comment (BasThomas) [inline MessageViewController/MessageView.swift]: We could even make the `switch` only get the button:

```swift
let button: UIButton
switch type {
case .left:
  button = leftButton
case .right
  button = rightButton
}
button.setImage(buttonIcon, for: state)
```

Comment (Fogh) [reply MessageViewController/MessageView.swift]: Good idea! I have implemented that 👍

**Comment by rnystrom** [comment]
Mind fixing up the conflicts?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
There shouldn’t be any?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Oh might be a GitHawk bug 😬 buttoned showed as conflicts

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
img_1915
Was green for me!

**Comment by rnystrom** [comment]
Ya it turned green for me too. It happened in another PR, something is off

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**